app/retail/templates/emails/template.html

```diff
@@ -323,6 +323,33 @@
     }
   }
   
+
+  .etherscan-link {
+    display: inline-block;
+    font-family: 'Muli', sans-serif;
+    color: #3E00FF;
+  }
+
+  .grant-txn-msg, .grant-txn-id {
+    margin: 0 auto;
+  }
+
+  .grant-txn-id a {
+    overflow-wrap: break-word;
+  }
+
+  @media (min-width: 768px) {
+    .grant-txn-msg {
+      width: 70%;
+    }
+  }
+
+  @media (min-width: 1200px) {
+    .grant-txn-msg {
+      width: 60%;
+    }
+  }
```

Comment:
^ could we move this to the needed email files! 
Else it would be imported by every mail

Reply:
@thelostone-mc Gmail seems to have an issue while using media queries in emails. It only considers the first set of media queries it sees and ignores media queries seen anywhere else afterwards. I found this behaviour while working on the daily email redesign and also found on the Internet that others have experienced this behaviour too.

The workaround is to include all the media queries in the first place (first <style> tag) and that would be the one in template.html. This is why the media queries and the corresponding styles are all placed in template.html itself.

Reply:
Okay that makes sense ! 
But if by placing it here, wouldn't every other email which has media query styles not be applied (not sure if there are other mails which get affected by this )

If we are doing this, we should collect it from all emails and add that in

Could we move these though into the grant specific mail ?
```
  .etherscan-link {
    display: inline-block;
    font-family: 'Muli', sans-serif;
    color: #3E00FF;
  }

  .grant-txn-msg, .grant-txn-id {
    margin: 0 auto;
  }

  .grant-txn-id a {
    overflow-wrap: break-word;
  }
```

Reply:
Okay that makes sense ! 
But if by placing it here, wouldn't every other email which has media query styles not be applied (not sure if there are other mails which get affected by this )

If we are doing this, we should collect it from all emails and add that in

Could we move these though into the grant specific mail ?
```
  .etherscan-link {
    display: inline-block;
    font-family: 'Muli', sans-serif;
    color: #3E00FF;
  }

  .grant-txn-msg, .grant-txn-id {
    margin: 0 auto;
  }

  .grant-txn-id a {
    overflow-wrap: break-word;
  }
```

Reply:
> Could we move these though into the grant specific mail ?
> 
> ```
>   .etherscan-link {
>     display: inline-block;
>     font-family: 'Muli', sans-serif;
>     color: #3E00FF;
>   }
> 
>   .grant-txn-msg, .grant-txn-id {
>     margin: 0 auto;
>   }
> 
>   .grant-txn-id a {
>     overflow-wrap: break-word;
>   }
> ```

@thelostone-mc Done in https://github.com/gitcoinco/web/pull/6791/commits/e444c77057a3ebcb39b4e2b78cc116288ef9479f

Reply:
> Okay that makes sense !
> But if by placing it here, wouldn't every other email which has media query styles not be applied (not sure if there are other mails which get affected by this )
> 
> If we are doing this, we should collect it from all emails and add that in

I do doubt that the media query styles present inside specific email templates are actually being rendered correctly by Gmail. That may have to be tested and taken care of. But the commits in this PR wont affect the other emails.